ms/MSSel/MSAntennaParse.h

```diff
@@ -174,6 +174,7 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
   public:
     static MSAntennaParse* thisMSAParser;
     static MSSelectionErrorHandler* thisMSAErrorHandler;
+    static void cleanupErrorHandler() {if (thisMSAErrorHandler) delete thisMSAErrorHandler;thisMSAErrorHandler=0x0;}
```

Comment:
please elaborate on how this makes debugging easier. If it were splitted into multiple lines I could see it, but not as written.

I'm not aware of coding style guidelines for casa, it would be good to get everything into a consistent shape but it would also be pretty annoying for people who are not used to others scrutinizing their code, it leads to lots of revisions before something gets merged.
But coding guidelines should not be discussed here, that should go into a new issue.